src/config.js

```diff
@@ -28,9 +28,10 @@ export const settings = {
 export const views = {
   ...defaultViews,
   contentTypesViews: {
+    training_class: TrainingView,
```

Comment:
This line must be after `...defaultViews.contentTypesViews` and to respect alphabetical order after `    presentation: PresentationView`